utils/build.py

```diff
@@ -91,6 +91,10 @@ def build(args):
     run(['ctest', '--output-on-failure'], args.builddir, env, args.dry_run)
 
 
+def cygpath(f):
+    p = subprocess.Popen(['cygpath', '-w', f], stdout=subprocess.PIPE)
```

Comment:
Can we use check_output here instead the output will be more clear. (I can't remember if this was failing on windows, if so, ignore my comment.)